### PR TITLE
README - Programmatic Usage

### PR DESCRIPTION
**Pull request by brdlyptrs**
Updates README to add section about interaction with hCaptcha API in a programmatic way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,44 @@ import { default as RenamedCaptcha } from '../utils/captcha';
 </FormComponent>
 ```
 
+#### Programmatic Usage
+In the event you want to call the hCaptcha client API directly you can do so by using the hook `useRef` and waiting for `onLoad` to be called. By waiting for `onLoad` the hCaptcha API will be ready and the hCaptcha client will have been setup. See the following example:
```
Comment (e271828-) [inline README.md:51]:
```suggestion
In the event you want to call the hCaptcha client API directly, you can do so by using the hook `useRef` and waiting for `onLoad` to be called. By waiting for `onLoad` the hCaptcha API will be ready and the hCaptcha client will have been setup. See the following example:
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,44 @@ import { default as RenamedCaptcha } from '../utils/captcha';
...
+```js
+import { useEffect, useRef, useState } from "react";
+import HCaptcha from "@hcaptcha/react-hcaptcha";
+
+export default function Form() {
+  const [token, setToken] = useState(null);
+  const captchaRef = useRef(null);
+
+  const onLoad = () => {
+    // this reaches out to the hcaptcha library and runs the
```
Comment (e271828-) [inline README.md:62]:
```suggestion
    // this reaches out to the hCaptcha JS API and runs the
```

File: README.md [hunk 3]
```diff
@@ -47,6 +47,44 @@ import { default as RenamedCaptcha } from '../utils/captcha';
...
+    // execute function on it. you can use other functions as well
```
Comment (e271828-) [inline README.md:63]:
```suggestion
    // execute function on it. you can use other functions as
```

File: README.md [hunk 4]
```diff
@@ -47,6 +47,44 @@ import { default as RenamedCaptcha } from '../utils/captcha';
...
+    // documented in the api:
```
Comment (e271828-) [inline README.md:64]:
```suggestion
    // documented here:
```

**Comment by brdlyptrs** [comment]
Looks good, thanks for the copy updates @e271828-

**Review by e271828- — APPROVED** [review]

**Pull request merged**